File: dataset/output-bk/MISTRAL_codestral-latest/Task149_MISTRAL_codestral-latest.java
import java.util.LinkedList;
import java.util.Queue;

public class MyQueue {
    private Queue<Integer> queue = new LinkedList<>();

    public void enqueue(int item) {
        queue.add(item);
    }

    public Integer dequeue() {
        if (queue.isEmpty()) {
            return null;
        }
        return queue.poll();
    }

    public Integer peek() {
        if (queue.isEmpty()) {
            return null;
        }
        return queue.peek();
    }
}

File: dataset/output-bk/MISTRAL_codestral-latest/Task149_MISTRAL_codestral-latest.cpp
#include <queue>

class Queue {
    private:
        std::queue<int> queue;

    public:
        void enqueue(int item) {
            queue.push(item);
        }

        int dequeue() {
            if (queue.empty()) {
                return -1;
            }
            int item = queue.front();
            queue.pop();
            return item;
        }

        int peek() {
            if (queue.empty()) {
                return -1;
            }
            return queue.front();
        }
};